scripts/globals/mobskills/seismostomp.lua

```diff
@@ -25,20 +24,16 @@ function onMobWeaponSkill(target, mob, skill)
     end
 
     local info = MobPhysicalMove(mob, target, skill, numhits, accmod, dmgmod, TP_NO_EFFECT)
-
-    local shadows = info.hitslanded
-
-    -- wipe shadows
-    if (mob:isMobType(MOBTYPE_NOTORIOUS)) then
-        shadows = MOBPARAM_WIPE_SHADOWS
+    
+    if math.random(2) == 1 then
+    local dmg = MobFinalAdjustments(info.dmg, mob, skill, target, tpz.attackType.PHYSICAL, tpz.damageType.BLUNT, MOBPARAM_1_SHADOW)
+    
+    else
```

Comment:
indents were off and I dislike seeing the same var local'd twice so I was about to suggest we just us a variable for the shadows in a conditional instead of copying the function call into conditionals, but then I realized the shadow values are actually 1 and 2 for the MOBPARAM https://github.com/project-topaz/topaz/blob/27b97b7bf30b8336ca5a0a7b9117ce7eed205f18/scripts/globals/monstertpmoves.lua#L27

So this does the same work, but slightly cleaner to evaluate:
```lua
function onMobWeaponSkill(target, mob, skill)
    local numhits = 1
    local accmod = 1
    local dmgmod = 2.3

    if mob:isMobType(MOBTYPE_NOTORIOUS) then
        dmgmod = dmgmod + math.random()
    end

    local info = MobPhysicalMove(mob, target, skill, numhits, accmod, dmgmod, TP_NO_EFFECT)
    local dmg = MobFinalAdjustments(info.dmg, mob, skill, target, tpz.attackType.PHYSICAL, tpz.damageType.BLUNT, math.random(2))
    local typeEffect = tpz.effect.STUN

    MobPhysicalStatusEffectMove(mob, target, skill, typeEffect, 1, 0, 4)
    target:takeDamage(dmg, mob, tpz.attackType.PHYSICAL, tpz.damageType.BLUNT)
    return dmg
end
```